### ISSUE TITLE
Upper case "CREATE" in column name ends table parse

### ISSUE THREAD

**Issue by n2ygk** [xnuinside/simple-ddl-parser#94]
**Describe the bug**
It looks like you are seeing CREATE and breaking the table DDL parse...

**To Reproduce**

t1.sql:
```sql
DROP TABLE IF EXISTS demo;
--
CREATE TABLE demo
(
     foo                             char(1),
     create_date                     DATETIME2,
     created_BY                      VARCHAR (20)
)
--
```
t2.sql:
```sql
DROP TABLE IF EXISTS demo;
--
CREATE TABLE demo
(
     foo                             char(1),
     CREATE_date                     DATETIME2,
     created_by                      VARCHAR (20),
)
--
```
```
sdp -v t1.sql
$ sdp -v t1.sql
Start parsing file t1.sql 

File with result was saved to >> schemas folder
[{'alter': {},
  'checks': [],
  'columns': [{'check': None,
               'default': None,
               'name': 'foo',
               'nullable': True,
               'references': None,
               'size': 1,
               'type': 'char',
               'unique': False},
              {'check': None,
               'default': None,
               'name': 'create_date',
               'nullable': True,
               'references': None,
               'size': None,
               'type': 'DATETIME2',
               'unique': False},
              {'check': None,
               'default': None,
               'name': 'created_BY',
               'nullable': True,
               'references': None,
               'size': 20,
               'type': 'VARCHAR',
               'unique': False}],
  'index': [],
  'partitioned_by': [],
  'primary_key': [],
  'schema': None,
  'table_name': 'demo',
  'tablespace': None}]
```

vs:
```
$ sdp -v t2.sql
Start parsing file t2.sql 

File with result was saved to >> schemas folder
[{'alter': {},
  'checks': [],
  'columns': [{'check': None,
               'default': None,
               'name': 'foo',
               'nullable': True,
               'references': None,
               'size': 1,
               'type': 'char',
               'unique': False}],
  'index': [],
  'partitioned_by': [],
  'primary_key': [],
  'schema': None,
  'table_name': 'demo',
  'tablespace': None}]
```
**Expected behavior**
A capitalized substring field name that is CREATE shouldn't break the parser.

**Comment by n2ygk** [comment]
Looks like your tokenizer is misidentifying keywords even when they are no separated by white space.

**Comment by xnuinside** [comment]
@n2ygk no, ptoblem not in tokenizer, lol, problem in changes that I did in 0.22.0 version to get possible parse statements without ; or GO. I will fix it tomorrow and release the patch version. thanks for catching this

**Comment by n2ygk** [comment]
Haha so it was my fault :-)

**Comment by xnuinside** [comment]
@n2ygk fixed  in 0.22.2, test added: https://github.com/xnuinside/simple-ddl-parser/pull/95

**Comment by xnuinside** [comment]
@n2ygk Alan, I will close the issue - check my comment in PR) it was already released at the morning
